scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -850,6 +854,13 @@ def validate_model_spec(asset_config: assets.AssetConfig) -> int:
         _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.LICENSE} missing")
         error_count += 1
 
+    if MLFlowModelTags.HIDDEN_LAYERS_SCANNED not in model.tags:

```

Comment:
for consistency I think this might be better. Could you also please add in the logging that the hiddenlayertags is missing so devs know how to fix the error? Thank you!
```suggestion
    if not model.tags.get(MLFlowModelTags.HIDDEN_LAYERS_SCANNED)

```

Reply:
- Value for `hiddenlayerscanned` tag is set to `""` and  `model.tags.get(MLFlowModelTags.HIDDEN_LAYERS_SCANNED)` would be false even for scenario when tag exists. Hence checking if tag actually exist in the model spec. 
- updated logs to print missing tag

Reply:
ah yes, thanks!